src/mrpro/data/_KTrajectory.py

```diff
@@ -41,11 +43,99 @@ class KTrajectory:
 
     @property
     def broadcasted_shape(self) -> tuple[int, ...]:
-        """The broadcasted shape of the trajectory."""
+        """The broadcasted shape of the trajectory.
+
+        Returns
+        -------
+            broadcasted shape of trajectory
+        """
         shape = np.broadcast_shapes(self.kx.shape, self.ky.shape, self.kz.shape)
         return tuple(shape)
 
-    def as_tensor(self, stack_dim=0):
+    def _type_of_traj(
```

Comment:
could this instead by
`_type(self, dim=Literal{kx,ky,kz,k0,k1,k2], tolerance:float)`
as all dimensions are checked in a loop either way. this would remove the orientation string